test/parseForm.test.ts

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License.
+
+import { expect } from 'chai';
+import 'mocha';
+import { parseForm } from '../src/parsers/parseForm';
+
+describe('parseForm', () => {
+    describe('multipart', () => {
+        const contentType = 'multipart/form-data; boundary=----WebKitFormBoundaryeJGMO2YP65ZZXRmv';
```

Comment:
Should we define a const for the string "`WebKitFormBoundaryeJGMO2YP65ZZXRmv`" ? It's repeated multiple times.

Reply:
If it was for production code I would say yes, but for test code I often prefer to hard-code the test data even if there's duplicates. It just feels closer to the real deal when the inputs/outputs of the test are simple values without extra manipulation. Sometimes (but not always) I find it more readable this way, too.